src/app/containers/BulletedList/__snapshots__/index.test.jsx.snap

```diff
@@ -11,11 +11,18 @@ exports[`BulletedListContainer should render ltr correctly 1`] = `
   list-style-type: none;
 }
 
+.c2 > li {
+  position: relative;
+}
+
 .c2 > li::before {
-  content: ' ';
-  padding-left: 1.75rem;
-  background: url('data:image/svg+xml,%3Csvg%20height%3D%2710%27%20width%3D%2710%27%20xmlns%3D%27http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%27%3E%3Cstyle%3E%40media%20screen%20and%20%28-ms-high-contrast%3A%20active%29%7Bsvg%7Bfill%3A%20%23fff%7D%7D%3C%2Fstyle%3E%3Ccircle%20cx%3D%275%27%20cy%3D%275%27%20r%3D%273%27%3E%3C%2Fcircle%3E%3C%2Fsvg%3E') no-repeat center;
```

Comment:
@rhenshaw56 do you know what these backgrounds were for?

Reply:
These are the bulleted items in bulleted lists - this is now in CSS instead of an SVG